test/parallel/test-crypto-random.js

```diff
@@ -315,3 +315,21 @@ assert.throws(
   assert.strictEqual(desc.writable, true);
   assert.strictEqual(desc.enumerable, false);
 });
+
+
+{
+  const randomInts = [];
+  for (let i = 0; i < 100; i++) {
+    crypto.randomInt(1, 4, common.mustCall((err, n) => {
+      assert.ifError(err);
+      assert.ok(n >= 1);
+      assert.ok(n < 4);
+      randomInts.push(n);
+      if (randomInts.length === 100) {
+        assert.ok(randomInts.includes(1));
+        assert.ok(randomInts.includes(2));
+        assert.ok(randomInts.includes(3));
+      }
+    }));
```

Comment:
You should also test negative integers etc.

Reply:
Added a couple of tests.